orchestration/dagster_orchestration/hca_orchestration/solids/validate_egress.py

```diff
@@ -74,6 +74,7 @@ def base_notify_slack_of_egress_validation_results(
             f"Problems identified in post-validation for HCA {gcp_env} dataset {dataset_name}:",
             "Duplicate lines found: " + str(validation_results.duplicates),
             "Null file references found: " + str(validation_results.null_file_refs),
+            "Dangling project references found: " + str(validation_results.dangling_project_refs)
```

Comment:
i can't tag the line in github's ui, but you'll also need to update line 72 to check if there are dangling project references in the first place.

given that there are now three checks like this, i think it might be worth wrapping `ProblemCount` in a class or at least delegating "is this validation result successful" to a helper method, to make that line a little easier to parse

Reply:
Good call, @ThePletch check out https://github.com/DataBiosphere/hca-ingest/pull/81/commits/0bc3127c1ec2be8625d91a9818e6d5541a5320a8